### ISSUE TITLE
Add Python 2.6

### ISSUE THREAD

**Issue by Symbian9** [niess/python-appimage#38]
Please, add also Python 2.6.

**Comment by niess** [comment]
Hello,

this project builds Python AppImages from the [PyPa/manylinux](https://github.com/pypa/manylinux) Docker images. Python 2.6 is not part of the manylinux images. Thus, it cannot be added as so.

A workaround might be to build a new Docker image on top of manylinux1 containing Python2.6. Then `python-appimage` should be able to generate a Python2.6 appimage from this modified Docker image.

**Comment by Symbian9** [comment]
> A workaround might be to build a new Docker image on top of manylinux1

Yes, please! As there are a lot of software stuck in Python 2.6 (e.g. *Blender* <= 2.49b, etc.).

**Comment by niess** [comment]
I see. 

If you can provide me with this modified Docker image I would be glad to add / update `python-appimage` accordingly with Python 2.6.

Note that python2.6 would be installed to /opt/python in the Docker image, as other distributions. Then building the AppImage would work out of the box. I might be too optimistic though :P

**Comment by Symbian9** [comment]
> If you can provide me with this modified Docker image

I has no such.
